### PR TITLE
Generator names are too long. Humanize it

### LINKED ISSUE #68: Long names break generators grid
yeoman-app-padding-bottom

### System details:
* Ubuntu 15.04
* Node: v0.12.4

### Console logs:
```
App load time: 221ms
[3878:0807/205758:INFO:CONSOLE(109)] "Download the React DevTools for a better development experience: https://fb.me/react-devtools", source: /home/danilo/Desenvolvimento/nodejs/yeoman-app/node_modules/react/lib/React.js (109)
[3878:0807/205758:INFO:CONSOLE(48)] "Warning: Failed propType: Invalid prop `openOnMount` of type `string` supplied to `Snackbar`, expected `boolean`. Check the render method of `CompatibilityWarning`.", source: /home/danilo/Desenvolvimento/nodejs/yeoman-app/node_modules/react/lib/warning.js (48)
[3878:0807/205758:INFO:CONSOLE(0)] "Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check http://xhr.spec.whatwg.org/.", source:  (0)
[3878:0807/205758:INFO:CONSOLE(775)] "rendered file:///home/danilo/Desenvolvimento/nodejs/yeoman-app/static/less/main.less successfully.", source: file:///home/danilo/Desenvolvimento/nodejs/yeoman-app/node_modules/less/dist/less.js (775)
[3878:0807/205758:INFO:CONSOLE(775)] "css for file:///home/danilo/Desenvolvimento/nodejs/yeoman-app/static/less/main.less generated in 196ms", source: file:///home/danilo/Desenvolvimento/nodejs/yeoman-app/node_modules/less/dist/less.js (775)
[3878:0807/205758:INFO:CONSOLE(775)] "less has finished. css generated in 196ms", source: file:///home/danilo/Desenvolvimento/nodejs/yeoman-app/node_modules/less/dist/less.js (775)
[3897:0807/205758:ERROR:gles2_cmd_decoder.cc(11539)] [GroupMarkerNotSet(crbug.com/242999)!:D0A8DCCE41350000]GL ERROR :GL_INVALID_OPERATION : glTexStorage2DEXT: <- error from previous GL command
```

### PR DESCRIPTION
**Pull request by bitshadow**
Fixes #68 

I have added new module `underscore.string`. Let me know if you want to add separate util method instead of full module.

### CODE REVIEW

**Comment by ruyadorno** [comment]
hmmmm, not sure about adding yet another dependency just for that... we already have `underscore-plus` that has a [similar method](https://github.com/atom/underscore-plus/blob/master/src/underscore-plus.coffee#L142) that can be used instead

File: src/renderer/components/grid.jsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ var React = require('react');
 var color = require('color');
 var mui = require('material-ui');
 var classSet = require('classnames');
+var humanize = require('underscore.string/humanize');
```
Comment (ruyadorno) [inline src/renderer/components/grid.jsx]: maybe just:

```js
var humanize = require('underscore-plus').humanizeEventName;
```

instead

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "season": "^5.3.0",
     "semver": "^5.0.1",
     "underscore-plus": "^1.6.1",
+    "underscore.string": "^3.1.1",
```
Comment (ruyadorno) [inline package.json]: no need for the extra dep

**Comment by bitshadow** [comment]
Thanks @ruyadorno. Fixed it.

**Comment by ruyadorno** [comment]
np :blush: not sure only this will fix #68 though, have you tested it with a really long generator name?

**Comment by bitshadow** [comment]
Longest I tried `angular-crud-custom-module` which comes under single line.

If we want to prevent multiline behaviour then either we can reduce the font size or use `text-overflow: ellipsis`. what do you think?

**Comment by ruyadorno** [comment]
I see, just by stripping the `generator-` prefix we already save a lot of space!

Can you please implement `text-overflow: ellipsis` just for the rare cases of a very very long name?

Just this and let's merge!

**Comment by bitshadow** [comment]
@ruyadorno  I see lessCompiler is loaded in the main.js but my `less` changes are not reflecting in UI. Any idea?

**Comment by ruyadorno** [comment]
hmmmm... not really :disappointed: in my latest changes I started doing all styling inline, I have it in my plans to get rid of the `less` setup in favor of inline styling evetually

**Comment by ruyadorno** [comment]
so I would recommend you to add this style inline too :blush:

**Comment by bitshadow** [comment]
That worked :+1:. Let me know if further changes required.

**Comment by ruyadorno** [comment]
:confetti_ball: awesome!

many thanks @bitshadow

**Pull request merged**